sql/core/src/test/resources/sql-tests/results/group-by.sql.out

```diff
@@ -355,10 +354,9 @@ struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "_LEGACY_ERROR_TEMP_2422",
+  "errorClass" : "AGGREGATION_WITHOUT_GROUP_BY",
   "messageParameters" : {
-    "aggExprs" : "()",
-    "sqlExpr" : "id"
+    "aggExprs" : ""
```

Comment:
Here is the case when there are no aggregations at all:
```sql
SELECT id FROM range(10) HAVING id > 0
```
How about just `MISSING_GROUP_BY`?